tests/integration/zblock_mount_unmount_test.go

```diff
@@ -95,9 +95,10 @@ func (s *BlockMountUnMountSuite) setupPVCs() {
 	logger.Infof("creating the test PVCs")
 	poolNameRWO := "block-pool-rwo"
 	storageClassNameRWO := "rook-ceph-block-rwo"
+	reclaimPolicy := "Delete"
```

Comment:
Setting this as a variable isn't needed as there is only one usage.

Reply:
@galexrt Here are the tests for `Retain` reclaimPolicy: https://github.com/rook/rook/pull/1995/files#diff-e8a82bc6f9bb7f8be12d6695a2df1fdeR45